### ISSUE TITLE
[feature request] Avoid environment variable based logic

### ISSUE THREAD

**Issue by gaocegege** [kubeflow/fate-operator#12]
Now we have the logic as follows:

```go
	serviceurl := fmt.Sprintf("%s-kubefate-%s.%s:8080", PREFIX, kubefateCR.Name, kubefateCR.Namespace)
	if os.Getenv("FATE_OPERATOR_TEST") == "true" {
		serviceurl = kubefateCR.Spec.IngressDomain
	}
```

It is hard to maintain. Prefer to have a CLI argument to implement it.

**Comment by LaynePeng** [comment]
@owlet42 please have a look into it.

**Comment by LaynePeng** [comment]
/close
Close cuz fix on pr#18
